### PR TITLE
Disable bios interfaces idrac-wsman

### PR DESCRIPTION
**Pull request by kashifest**
A temporary fix for the issue #188 

We can enable idrac once we have done proper investigation.

### CODE REVIEW

**Comment by fmuyassarov** [comment]
I think it failed because CI didn't build ironic-image but instead pulled it from the quay.

**Comment by kashifest** [comment]
> I think it failed because CI didn't build ironic-image but instead pulled it from the quay.

Yes, there is another bug in metal3-dev-env. Thats why the previous PR also went it without giving the error.

**Comment by fmuyassarov** [comment]
I wondering why if we need the CI here since the images are pulled from the public registry instead of building and testing them locally.

**Comment by furkatgofurov7** [comment]
As #162  causing the problem in CI currently, there is another PR #181 that might cause the CI fail again maybe once it is in.

**Comment by kashifest** [comment]
@furkatgofurov7 those are not enabling bios interfaces, so lets see , we need to fix  metal3-dev-env to make sure that the local ironic image is tested properly in CI, I beleive @maelk is looking into it

**Comment by maelk** [comment]
The metal3-dev-env fix is here : https://github.com/metal3-io/metal3-dev-env/pull/420

File: ironic.conf [hunk 1]
```diff
@@ -5,7 +5,6 @@ default_boot_interface = ipxe
 default_deploy_interface = direct
 default_inspect_interface = inspector
 default_network_interface = noop
-enabled_bios_interfaces = idrac-wsman
```
Comment (dtantsur) [inline ironic.conf:8]: The correct fix would be to add `no-bios` here.

Comment (furkatgofurov7) [reply ironic.conf:8]: @dtantsur should we open a PR with the configuration you are suggesting?

**Pull request merged**